src/data/crops/garlic.js

```diff
@@ -0,0 +1,29 @@
+import { crop, fromSeed } from '../crop'
+import { cropLifeStage, cropType } from '../../enums'
+
+const { SEED, GROWING } = cropLifeStage
+
+/**
+ * @property farmhand.module:items.garlicSeed
+ * @type {farmhand.item}
+ */
+export const garlicSeed = crop({
+  cropType: cropType.GARLIC,
+  cropTimetable: {
+    [SEED]: 2,
+    [GROWING]: 3,
+  },
+  growsInto: 'garlic',
+  id: 'garlic-seed',
+  name: 'Garlic Bulb',
+  tier: 3,
```

Comment:
If we change this to be unlocked at a later level like 34 (see my comment in `levels.js`), it would make sense to increase `tier` to 5, as it would be the third tier 5 seed. Having about three items per tier (unlocked in order) seems to be a good balance so far.